src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs

```diff
@@ -11,6 +11,39 @@
 
 namespace OutGridView.Cmdlet
 {
+    public class ConsoleGuiWindow : Window
```

Comment:
Potentially worth going in its own file.

Reply:
It'll go away once we switch to `StatusBar`.